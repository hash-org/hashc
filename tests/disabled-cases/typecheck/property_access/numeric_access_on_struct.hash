// run=fail
Frobulate := struct(
  data: i32,
  other: char = 'c',
)

t := Frobulate(data = 23)


main := () -> i32 => {
  t.2
}
