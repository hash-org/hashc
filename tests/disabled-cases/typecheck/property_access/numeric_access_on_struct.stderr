error[0015]: the field `2` is not present within `Frobulate(data = 23_i32, other = 'c')`
  --> $DIR/numeric_access_on_struct.hash:11:3
10 |   main := () -> i32 => {
11 |     t.2
   |     ^ `Frobulate(data = 23_i32, other = 'c')` does not contain the property `2`
12 |   }
